python/hail/methods/qc.py

```diff
@@ -480,7 +480,12 @@ def vep(dataset: Union[Table, MatrixTable], config, block_size=1000, name='vep',
 
     **Annotations**
 
-    A new row field is added in the location specified by `name` with type given by the type given by the `json_vep_schema` (if `csq` is ``False``) or :py:data:`.tstr` (if `csq` is ``True``).
+    A new row field is added in the location specified by `name` with type given
+    by the type given by the `json_vep_schema` (if `csq` is ``False``) or
+    :py:data:`.tstr` (if `csq` is ``True``).
+
+    If csq is ``True``, then the CSQ header string is also added as a global
```

Comment:
what happens if `name + '_csq_header'` already exists? Should we have a nicer error message? Warning that we're overwriting that global field?

I see from below that this returns a new VEP table, so there will not be any conflicts with the global path. However, my comment still stands that there is a mismatch between the docs and code with regards to the path name.